notebooks.go

```diff
@@ -1138,20 +1052,18 @@ func (s *server) GetNotebookEvents(w http.ResponseWriter, r *http.Request) {
 
 	log.Printf("getting events in %q in %q", notebook, namespace)
 
-	eventOpts := metav1.ListOptions{
-		FieldSelector: "involvedObject.kind=Notebook,involvedObject.name=" + notebook,
-	}
-	events, err := s.clientsets.kubernetes.CoreV1().Events(namespace).List(context.TODO(), eventOpts)
+	events, err := s.listEvents(namespace, "Notebook", notebook)
 	if err != nil {
-		log.Printf("failed to load events for %s/%s: %v", namespace, notebook, err)
+		s.error(w, r, err)
+		return
 	}
 
 	resp := &notebookeventsresponse{
 		APIResponseBase: APIResponseBase{
 			Success: true,
 			Status:  http.StatusOK,
 		},
-		Events: events.Items,
+		Events: events,
```

Comment:
does this change the 'reception' anywhere? I could not find what items used to be.

Reply:
it relates to your previous comment. events.items was a []events, this events is now a []*events. because the way to retrieve that data changes. 
Has no impact based on my testing